playwright-tests/tests/community/discussions.spec.js

```diff
@@ -29,7 +29,10 @@ test.describe("Wallet is connected", () => {
     storageState: "playwright-tests/storage-states/wallet-connected-peter.json",
   });
 
-  test("should create a discussion when content matches", async ({ page }) => {
+  // Skipping this test because we might disable discussions altogether
+  test.skip("should create a discussion when content matches", async ({
```

Comment:
As long as the discussions are using the old indexer, we should still keep this test

Reply:
resolved, it is because of an RPC timeout. I added it back in.